src/main/java/org/opensearch/security/rest/SecurityWhoAmIAction.java

```diff
@@ -111,7 +111,7 @@ public void accept(RestChannel channel) throws Exception {
 
                         final String dn = sslInfo.getPrincipal();
                         final boolean isAdmin = adminDns.isAdminDN(dn);
-                        final boolean isNodeCertificateRequest = dn != null && WildcardMatcher.from(nodesDn, true).matchAny(dn);
+                        final boolean isNodeCertificateRequest = dn != null && WildcardMatcher.from(nodesDn).ignoreCase().matchAny(dn);
```

Comment:
@nibix we don't have to call `.ignoreCase()` here right ?

Reply:
At this place, it is possibly not so important, as this code serves no security critical function. The security critical checks are performed in DefaultInterClusterRequestEvaluator where they are done with `ignoreCase()` as well. This would indicate that in the action it should be done in the same fashion.